newIDE/app/src/Export/ExportDialog.js

```diff
@@ -211,23 +243,11 @@ export default class ExportDialog extends React.Component<Props, State> {
                             )}
                       </List>
                       <Line justifyContent="center" alignItems="center">
-                        {!showExperimental ? (
-                          <FlatButton
-                            key="toggle-experimental"
-                            icon={<Visibility />}
-                            primary={false}
-                            onClick={() => this._showExperimental(true)}
-                            label={<Trans>Show experimental exports</Trans>}
-                          />
-                        ) : (
-                          <FlatButton
-                            key="toggle-experimental"
-                            icon={<VisibilityOff />}
-                            primary={false}
-                            onClick={() => this._showExperimental(false)}
-                            label={<Trans>Hide experimental exports</Trans>}
-                          />
-                        )}
+                        <ExperimentalExportButton
+                          showExperimental={showExperimental}
+                          onClick={value => this._showExperimental(value)}
+                          disabled
+                        />
```

Comment:
Extracted this out, as at the moment, cocos2d was the only experimental export. If in future we decide to add more experimental exports, this should come in handy.